src/translations/ru.tsx

```diff
@@ -6,10 +6,15 @@ const ru = {
   "no info": "неизвестно",
   Period: "Месячные",
   today: "сегодня",
+  "Today is the": "Сегодня",
```

Comment:
You can delete this and more because of the comment above.

Reply:
Has been deleted